src/documents/components/helpers/DocumentListHelp.js

```diff
@@ -2,20 +2,23 @@ import React from 'react';
 import BaseHelper from './BaseHelper';
 import addDocuments from '../../../assets/add_file.svg';
 
-const Help = () => (
+const Help = ({history}) => (
   <BaseHelper title="About Study Documents" image={addDocuments}>
-    <p>
-      Many documents need to be collected in order to properly ingest a study
-      into the Kids First Data Resource Center (DRC). Required documents may be
-      uploaded to a study here so that they may be reviewed and processed by the
-      Kids First DRC.
-    </p>
-    <p>
-      <b>Versions</b> allow revisions of the same document to be tracked.
-      Creation of a new document that contains revised data of another document
-      should be avoided and instead uploaded as a new version of the existing
-      document.
-    </p>
+    {history.location.pathname.includes('SF_') ? (
```

Comment:
Can you instead split these into different helpers and render them via the study router?